### PR TITLE
Add es5 to 6 conversion script

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- adds a script to automate es5 to 6 conversion
- adds a conversion assignment list by github user (only platform team for the moment)

## What is the value of this and can you measure success?

adds the script so the platform team can battle test it

### CODE REVIEW

File: tools/es5to6-files-per-user.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+const misc = [
+    'desbo',
+    'akash1810',
+    'dominickendrick',
+    'davidfurey',
+    'jranks123',
+    'ShaunYearStrong',
+    'jfsoul',
+    'gustavpursche',
+];
```
Comment (SiAdcock) [inline tools/es5to6-files-per-user.js]: Why are these humans `misc`? 😄

Comment (SiAdcock) [inline tools/es5to6-files-per-user.js]: Oh I see, you've assigned them some misc files to convert

Comment (sndrs) [inline tools/es5to6-files-per-user.js]: ha! yeah

**Review by SiAdcock — APPROVED** [review]
Let's do this!

File: tools/es5to6-files-per-user.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+const platformFilesPerHuman = filesPerHuman(platformModules, platform);
+
+const misc = [
+    'desbo',
+    'akash1810',
+    'dominickendrick',
+    'davidfurey',
+    'jranks123',
+    'ShaunYearStrong',
+    'jfsoul',
```
Comment (NataliaLKB) [inline tools/es5to6-files-per-user.js]: Why are Jon and Gustav part of the misc, and not part of platform? They are very much part of the platform team 😄

**Pull request merged**